ui/contexts/metamask-notifications/metamask-notifications.tsx

```diff
@@ -45,6 +46,8 @@ export const MetamaskNotificationsProvider: React.FC = ({ children }) => {
     [isProfileSyncingEnabled, isNotificationsEnabled],
   );
 
+  useAccountSyncingEffect();
```

Comment:
CLEAN!

(Future) - I don't like this context, I'd rather we replace all this with hooks too.